ciao-cli/instance.go

```diff
@@ -519,9 +519,14 @@ func (cmd *instanceAddCommand) run(args []string) error {
 		fatalf(err.Error())
 	}
 
+	if len(servers.Servers) < cmd.instances {
```

Comment:
And there is also the expectation that there would be as many lines as requested instances whereas there might be fewer. I'll look at addressing that.